Sources/Sugar/Helpers/Model.swift

```diff
@@ -11,7 +11,22 @@ public extension Model {
             .unwrap(or: Abort(.notFound, reason: "\(Self.self) with id \(id) not found"))
     }
 
-    func saveOrUpdate (
+    @available(*, deprecated, message: "use `creatOrUpdate(given:withSoftDeleted:restore:on)`")
+    func saveOrUpdate(
+        given filters: [FilterOperator<Self.Database, Self>],
+        withSoftDeleted: Bool = false,
+        restore: Bool = false,
+        on db: DatabaseConnectable
+    ) throws -> Future<Self> {
+        return try createOrUpdate(
```

Comment:
**return** might be removed.

Reply:
Would be a breaking change for project depending on this that don't run swift 5.1